data/soapvalidator/Mail/Bugs/ZBUG-3364.xml

```diff
@@ -0,0 +1,516 @@
+    <t:test id="GetMsgRequest">
+       <t:request>
+            <GetMsgRequest xmlns="urn:zimbraMail">
+                <m id="${message.id3}"/>
+            </GetMsgRequest>
+       </t:request>
+       <t:response>
+			<t:select path="//mail:GetMsgResponse/mail:m[@id='${message.id3}']">
+            <t:select path="//mail:e[@t='f']" attr="a" match="${test_list2.name}"/>
+            <t:select path="//mail:e[@t='t']" attr="a" match="${test_account4.name}"/>
+            <t:select path="//mail:su" match="${msg03.subject}"/>
+            </t:select>
+		</t:response>
+    </t:test>
+	
+	<!-- Create a persona for DL -->
+	 <t:test id="CreateIdentityRequest02" required="true">
+        <t:request>
+			<CreateIdentityRequest xmlns="urn:zimbraAccount">
+				<identity name="${persona02.name}">
+					<a name="zimbraPrefIdentityName">${persona02.name}</a>
+					<a name="zimbraPrefFromDisplay">${persona02.zimbraPrefFromDisplay}</a>
+					<a name="zimbraPrefFromAddress">${test_list2.name}</a>
+					<a name="zimbraPrefFromAddressType">sendOnBehalfOf</a>
+				</identity>
+			</CreateIdentityRequest>
+        </t:request>
+        <t:response>
+			<t:select path="//acct:CreateIdentityResponse/acct:identity" attr="name" set="persona02.name"/>
+			<t:select path="//acct:CreateIdentityResponse/acct:identity" attr="id" set="persona02.id"/>
+			<t:select path="//acct:CreateIdentityResponse/acct:identity/acct:a[@ name='zimbraPrefFromAddress']" match="${test_list2.name}"/>
+        </t:response>
+    </t:test>
+    
+    <!-- Send an email using persona and verify no error is encountered -->
+    <t:test id="sendmail1" required="true">
+		<t:request>
+			<SendMsgRequest xmlns="urn:zimbraMail">
+				<m idnt="${persona02.id}">
+					<e t="f" a="${test_list2.name}" p='${persona02.name}'/>
+					<e t="t" a="${test_account4.name}"/>
+					<su> ${msg03.subject}</su>
+					<mp ct="text/plain">
+					<content> ${message.content2}</content>
+					</mp>
+				</m>
+			</SendMsgRequest>
+		</t:request>
+		<t:response>
+			<t:select path="//zimbra:Code" match="service.PERM_DENIED" emptyset="1"/>
+			<t:select path="//mail:SendMsgResponse/mail:m" attr="id" set="message.id3"/>
+		</t:response>
+	</t:test>
+	</t:test_case>
+	
+	<!-- Verify message details using GetMsgRequest -->
+    <t:test id="GetMsgRequest">
+        <t:request>
+            <GetMsgRequest xmlns="urn:zimbraMail">
+                <m id="${message.id3}"/>
+            </GetMsgRequest>
+        </t:request>
+       <t:response>
+			<t:select path="//mail:GetMsgResponse/mail:m[@id='${message.id3}']">
+            <t:select path="//mail:e[@t='f']" attr="a" match="${test_list2.name}"/>
+            <t:select path="//mail:e[@t='f']" attr="p" match="${persona02.name}"/>
+            <t:select path="//mail:e[@t='t']" attr="a" match="${test_account4.name}"/>
+            <t:select path="//mail:su" match="${msg03.subject}"/>
+			</t:select>
+			</t:response>
+    </t:test>
+	
```

Comment:
Please add validation at recipient end to check from/to address and other message details